images/router/haproxy/conf/haproxy-config.template

```diff
@@ -39,12 +39,12 @@ defaults
   option httplog
   log global
 {{ end }}
+  timeout connect {{env "HAPROXY_DEFAULT_CONNECT_TIMEOUT" "5s"}}
```

Comment:
Nit: the naming convention for the environment variables all start with `ROUTER_*` so they could be changed to `ROUTER_DEFAULT_CONNECT_TIMEOUT` etc.